static/src/stylesheets/module/identity/_button.scss

```diff
@@ -88,6 +85,23 @@
         &.inline-cross {
             transform: scale(.5);
         }
+        &.inline-arrow-down {
+            transform: scale(.9);
+        }
+        &.inline-arrow-up {
+            transform: translateY(-10%) scale(.9);
+        }
+        &.inline-tick {
+            transform: scale(.8);
+        }
+        &.inline-envelope-add,
+        &.inline-envelope-remove {
+            transform: translateY(-5%) scale(.9);
```

Comment:
Question just for my understanding: could we do anything different with these scale numbers or are they just dictated by the icons provided? They are a bit magic to me.

Reply:
They are dictated by the icons sadly, as they have different visual weights and sizes and need some adjustments to look <a href="https://material.io/design/iconography/system-icons.html#grid-keyline-shapes">just right</a>. Something I normally like doing is to fit all icons into a uniform 1:1 box but that would require redoing them all